tests/bindings/golang/test_glalby.go

```diff
@@ -2,12 +2,12 @@ package main
 
 import (
 	"log"
-
-	"example.org/golang/glalby"
+	// "example.org/golang/glalby"
 )
 
 
 func main() {
-	log.Printf("equal: %v", glalby.Equal(1, 1))
-	log.Printf("equal: %v", glalby.Equal(0, 1))
+	log.Printf("TODO: test new methods")
```

Comment:
before merging, I will add some basic tests. Should expect a mnemonic to be passed to env like the examples do